### DISCUSSION TITLE
SO 2.3.200 broke my authentication

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#9612] [Unsupported Versions] [answered]
Hello!

After I have updated my distributed SO deployment to 2.3.200 I get the following error in SOC

![image](https://user-images.githubusercontent.com/81571847/214025042-5150fb12-50fd-42f2-b1f6-c6fba060fab0.png)

and can no longer login to Elasticsearch via Kibana (Username or password is incorrect). What happened here and how can I correct it? I already rebooted my Manager and the Search Nodes without success.

Cheers, Ben

**Comment by InfosecGoon** [comment, marked as answer]
What do you get as output from this command on the Manager?

sudo salt-call pillar.get elasticsearch:auth:enabled

**Reply by ben-sec** [reply]
True. It was an update from 190 to 200 and Elasticsearch authentication has been enabled previously.

**Reply by InfosecGoon** [reply]
Two things I would check, then:

1. The actual cause of the error in SOC should show up in /opt/so/log/sensoroni/sensoroni.log - if there's an issue with the Elasticsearch environment, it should show up there.
2. It's possible that your SOC accounts and Elastic accounts are out of sync, try running this from a root command prompt: "FORCE_SYNC=true so-user sync".

**Reply by ben-sec** [reply]
The FORCE_SYNC command did the trick. Thanks a lot!
